observer.php

```diff
@@ -0,0 +1,53 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+/**
+ * This file contains the page view if the user has the capability 'perform_observations'
+ *
+ * @package   mod_observation
+ * @copyright  2021 Endurer Solutions Team
+ * @author Matthew Hilton <mj.hilton@outlook.com>
+ * @license   http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+
+require('../../config.php');
+require('./classes/instructions.php');
+require('./classes/datalib.php');
+
+$id = required_param('id', PARAM_INT); // Observation instance ID.
+list($observation, $course, $cm) = get_observation_course_cm_from_obid($id);
+
+// Check permissions.
+require_login($course, true, $cm);
+require_capability('mod/observation:performobservation', $PAGE->context);
+
+// Render page.
+global $CFG, $PAGE, $OUTPUT;
```

Comment:
You shouldn't need the global declarations here. You are already in the global scope. You only need to declare this in subscopes like function scope or class scope